### DISCUSSION TITLE
How to open a new floating pane in a specific folder?

### DISCUSSION THREAD

**Discussion by morenohernan** [zellij-org/zellij#2746] [Q&A] [answered]
Hi, I've followed the docs here: https://zellij.dev/documentation/cli-actions

And I'm trying to open a new floating pane from another process (via neovim) in an specific path. So far when do this:
zellij action new-pane --floating --cwd "/Users/myuser/mypath" the pane is created but the prompt is with the default creation path(example my home dir) 

Can you please point out what is wrong with the command above?

**Comment by cristiand391** [comment]
afaik, that works when you specify a command to run in the new pane.

`zellij action new-pane --floating --cwd "/Users/myuser/mypath" ` doesn't pass any command after `--` so it will just inherit the cwd from the pane it was opened.

Example: run `bat package.json` in `~/code/gh/sf-plugin-api/` from `/home/cristiand391`:

![image](https://github.com/zellij-org/zellij/assets/6853656/524a7fa2-4c07-4c0b-94d9-9a1ee51be16e)

**Reply by cristiand391** [reply, marked as answer]
In your case you can pass the shell as the cmd:

`zellij action new-pane --floating --cwd "/Users/myuser/mypath" -- zsh`

**Comment by morenohernan** [comment]
Thanks a lot for the help @cristiand391  !!!! , it worked.
